{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -1,10 +1,17 @@
 [tool.poetry]
-name = "{{ cookiecutter.project_slug }}"
+name = "{{ cookiecutter.package_name }}"
 version = "0.1.0"
+packages = [{include = "{{ cookiecutter.package_name }}"}]
 description = ""
 authors = []
 readme = "README.md"
-packages = [{include = "lib"}]
+license = "MIT"
+repository = "https://github.com/mim-solutions/{{ cookiecutter.repo_name }}"
```

Comment:
Should we leave `mim-solutions` or have a username/organization name in the config?

Reply:
I thinks it's OK as it is now